common/test/test_lint.py

```diff
@@ -51,6 +51,8 @@ def test_with_pylint(self):
         # Pylint base command
         cmd = [
             'pylint',
+            # Make sure BIT modules can be improted (to detect "no-member")
+            '--init-hook=import sys;sys.path.insert(0, "./../qt");',
```

Comment:
This hook make pylint able to import our BIT modules and decide about "naming-errors".